internal/clients/instance_description.go

```diff
@@ -0,0 +1,13 @@
+package clients
+
+// InstanceInfo defines a model for an instance description
+type InstanceDescription struct {
+	// The id of the instance
+	ID *string `json:"id,omitempty" yaml:"id"`
+
+	// The public ipv4 dns of the instance
+	DNS *string `json:"dns,omitempty" yaml:"dns"`
+
+	// the public ipv4 of the instance
+	IPV4 *string `json:"ipv4,omitempty" yaml:"ipv4"`
+}
```

Comment:
I would suggest to name both flags "PublicXXX" just in case we want to add private info later on.

And maybe `IPv4` looks slightly better :-)

Just a thought - if IPv6 is also available, I would show it too. By default it will be likely empty because IPv6 you need to enable manually on AWS EC2.